components/centraldashboard/public/assets/i18n/languages.json

```diff
@@ -0,0 +1,60 @@
+{
+    "languages": {
+        "en": {
+            "dashboardView.headingQuickLinks": "Quick shortcuts",
+            "dashboardView.quicklinkUploadText": "Upload a pipeline",
+            "dashboardView.quicklinkUploadDesc": "Pipelines",
+            "dashboardView.quicklinkViewAllText": "View all pipeline runs",
+            "dashboardView.quicklinkViewAllDesc": "Pipelines",
+            "dashboardView.quicklinkCreateNewText": "Create a new Notebook server",
+            "dashboardView.quicklinkCreateNewDesc": "Notebook Servers",
+            "dashboardView.quicklinkViewKatibText": "View Katib Studies",
+            "dashboardView.quicklinkViewKatibDesc": "Katib",
+            "dashboardView.quicklinkMetadataArtifactsText": "View Metadata Artifacts",
+            "dashboardView.quicklinkMetadataArtifactsDesc": "Artifact Store",
+            "dashboardView.headingRecentPipelines": "Recent Pipelines",
+            "dashboardView.headingRecentPipelinesRuns": "Recent Pipeline Runs",
+            "dashboardView.headingDocumentation": "Documentation",
+            "dashboardView.docItemAAWDText": "FR Advanced Analytics Workspace Docs",
+            "dashboardView.docItemAAWDDesc": "FR Helpful guides about our data and analysis tools",
+            "dashboardView.docItemAAWDLink": "https://statcan.github.io/daaas/",
```

Comment:
* Avoiding all-caps in camel-case abbreviations makes them clearer.
* You have a structural pattern here with the three values per doc item.

So maybe...
```
"dashboardView.docItemAawDocs.text": "...",
"dashboardView.docItemAawDocs.desc": "...",
"dashboardView.docItemAawDocs.link": "...",
```

... and so on.